### ISSUE TITLE
Maintain Full Internal State

### ISSUE THREAD

**Issue by martindale** [FabricLabs/doorman#4]
When doorman is connected to a community, it should synchronize a full global state of members, messages, and topics.  Use the `maki-community` component.

**Comment by naterchrdsn** [comment]
so for the chat component, Slack has an internal state manager that it takes care of (Discord does not). Would we really need to duplicate this on our side, using memory for that?

**Comment by martindale** [comment]
Yes, and this seems largely implemented on latest `master` (and is in use on [`#idlerpg:verse.im`](https://chat.fabric.pub/#/room/#idlerpg:verse.im) — now need good tests and coverage for common events.
